tools/tapi/cmd/user.go

```diff
@@ -195,3 +211,27 @@ func userDelete(c *cli.Context) error {
 
 	return reportMessage(c, "User deleted.")
 }
+
+func userUpdatePassword(c *cli.Context) error {
+	userID := c.String(UserIDFlag)
+	password := c.String(PasswordFlag)
```

Comment:
I'm confused. This prompts for the user's current password, but does not appear to use it for anything?

Reply:
Good catch! 🤦‍♂